src/render/Cargo.toml

```diff
@@ -14,7 +14,7 @@
 
 [package]
 name = "gfx"
-version = "0.17.1"
+version = "0.17.2"
```

Comment:
we shouldn't touch this version, let's instead touch the `src/core` version

Reply:
core version is already bumped

Reply:
Done.